Triangle.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Describes triangles with double sides.
+ */
+class Triangle {
+  final int NUMBER_OF_SIDES = 3;
+  private double[] sides = new double[NUMBER_OF_SIDES];
+  
+  /**
+   * Triangle constructor; makes a triangle from the sequence of its sides.
+   * Sides of the created triangle are sorted by nondecreasing, even if the input sequence is not.
+   *
+   * @param sidesSequence a sequence of triangle sides
+   * @throws IllegalArgumentException if the sequence has not three sides.
+   * @throws NumberFormatException if sides are not numbers.
+   */
+  public Triangle(String[] sidesSequence) {
+    SequenceAnalyzer analyzer = new SequenceAnalyzer();
+    if (!analyzer.checkIfTriangleSides(sidesSequence)) {
```

Comment:
nope, SequenceAnalyzer must know nothing about triangle, only triangle by itself can know when it exists